runner.rb

```diff
@@ -0,0 +1,19 @@
+require './lib/stat_tracker'
+
+# game_path = './data/games.csv'
+# team_path = './data/teams.csv'
+# game_teams_path = './data/game_teams.csv'
+
+game_path = './data/games_test.csv'
+team_path = './data/teams_test.csv'
+game_teams_path = './data/game_teams_test.csv'
+
+locations = {
+  games: game_path,
+  teams: team_path,
+  game_teams: game_teams_path
+}
+
+stattracker = StatTracker.from_csv(locations)
```

Comment:
- change stattracker to stat_tracker (per Ruby convention)
- Do we want to keep stat_tracker with the regular data files, but include a stat_tracker_test as well so we can access both types of data?